### PR TITLE
support fetching on multiple topics in one request

### PR DESCRIPTION
**Pull request by evanmcc**
This is a rework of @tsloughter's work in #53, rebased and fixed up for some changes that have happened since then.

replaces #53

### CODE REVIEW

File: src/vg_client.erl [hunk 1]
```diff
@@ -47,25 +47,63 @@ ensure_topic(Pool, Topic) ->
...
+                  {ok, Pool} = vg_client_pool:get_pool(Topic, read),
+                  lager:debug("fetch request to pool: ~p ~p", [Topic, Pool]),
+                  case Acc of
+                      #{Pool := PoolReqs} ->
+                          Acc#{Pool => [R | PoolReqs]};
+                      _ ->
+                          Acc#{Pool => [R]}
+                  end
+          end, #{}, Requests),
+    %% should we do these in parallel?
```
Comment (tsloughter) [inline src/vg_client.erl:83]: Yea, but can leave this for now. Or do it simply by converting the map to a list and use `ec_plists:map`

File: config/sys.config [hunk 2]
```diff
@@ -1,17 +1,17 @@
 %% -*- erlang -*-
-[{vonnegut, [{chain, [{name, chain1},
+[{vonnegut, [%{chain, [{name, chain1},
```
Comment (tsloughter) [inline config/sys.config:2]: why is the chain commented out? or did I do that

**Review by tsloughter — APPROVED** [review]

File: config/sys.config (hunk 2 above)
Comment (evanmcc) [reply config/sys.config:2]: I think you did that.

Comment (tsloughter) [reply config/sys.config:2]: Well then, fuck it.

**Pull request closed without merging**